pages/account/[[...index]].tsx

```diff
@@ -198,27 +199,27 @@ export async function getServerSideProps({ req }) {
   // Parsing path from url to pass to ProfileTabs.
   const tabsPath = req.url.split("/", -1)[2] || null
   const id = patronTokenResponse.decodedPatron.sub
-  // try {
-  const { checkouts, holds, patron, fines } = await MyAccountFactory(id)
-  // Redirecting /fines if user has none.
-  if (tabsPath === "overdues" && fines.total === 0) {
+  try {
```

Comment:
I feel like I see this update in various PRs. Hope they can all be synced soon so there aren't many merge conflicts.

Reply:
Yes that's on me sorry . Will pull down a clean copy after this pr